### PR TITLE
fix(pip): added working dir to python setup.py exec

### PR DESCRIPTION
**Pull request by edena-legit**
Some `setup.py` files might load data from the file's working dir, such as the `README.md` file. Currently, when parsing a `setup.py` file and executing it, the Python command is executed without specifying the working dir.

### CODE REVIEW

**Comment by edena-legit** [comment]
@JamieMagee

File: src/Microsoft.ComponentDetection.Detectors/pip/PythonCommandService.cs [hunk 1]
```diff
@@ -57,10 +57,12 @@ private async Task<IList<string>> ParseSetupPyFileAsync(string filePath, string
             throw new PythonNotFoundException();
         }
 
+        var formattedFilePath = filePath.Replace('\\', '/');
```
[the remark this answers is not in the document]
Comment (cobya) [reply src/Microsoft.ComponentDetection.Detectors/pip/PythonCommandService.cs]: Thanks for the change!

**Comment by edena-legit** [comment]
@microsoft-github-policy-service agree company="Legit Security"

File: src/Microsoft.ComponentDetection.Common/PathUtilityService.cs [hunk 2]
```diff
@@ -75,4 +75,7 @@ public string ResolvePhysicalPath(string path)
     }
 
     private string ResolvePathFromInfo(FileSystemInfo info) => info.LinkTarget ?? info.FullName;
+
+    public string NormalizePath(string path) =>
```
Comment (melotic) [inline src/Microsoft.ComponentDetection.Common/PathUtilityService.cs:79]: nit: could we add tests for this?

**Comment by edena-legit** [comment]
@melotic anything else?

**Comment by cobya** [comment]
@edena-legit looks like the test run is failing with `NullReferenceException: Object reference not set to an instance of an object.` if you could please take a look

**Comment by edena-legit** [comment]
Hi, 

> @edena-legit looks like the test run is failing with `NullReferenceException: Object reference not set to an instance of an object.` if you could please take a look

Hi, thats weird as it seems it only occurs in windows build, it seems like runs after were ok..?

**Pull request closed without merging**